ui/app/src/components/ContentTypeManagement/components/TypeBuilderFormsEngine.tsx

```
@@ -165,87 +183,146 @@ function FieldBreadcrumbs(props: FieldFormViewProps): JSX.Element {
 	return (
 		fieldPathIds.length > 1 && (
 			<Breadcrumbs separator={<NavigateNextIcon fontSize="small" />}>
-				{fieldPathIds.map((id) => (
-					// TODO: Render field names instead of ids
-					<Typography variant="body2" key={id} children={id} />
-				))}
+				{fieldPathIds.map((id) => {
+					// Retrieve the fieldPathId by removing everything after `id` in fieldPathIds
+					const currentFieldPathId = fieldPathIds.slice(0, fieldPathIds.indexOf(id) + 1).join('.');
+					const currentField = getFieldFromType(props.type, currentFieldPathId);
+					return <Typography variant="body2" key={id} children={currentField.name} />;
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)